drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -215,30 +215,3 @@ void at86rf2xx_tx_exec(at86rf2xx_t *dev)
         netdev->event_callback(netdev, NETDEV2_EVENT_TX_STARTED, NULL);
     }
 }
-
-size_t at86rf2xx_rx_len(at86rf2xx_t *dev)
-{
-    uint8_t phr;
-
-    at86rf2xx_fb_read(dev, &phr, 1);
-
-    /* ignore MSB (refer p.80) and substract length of FCS field */
-    return (size_t)((phr & 0x7f) - 2);
-}
-
-void at86rf2xx_rx_read(at86rf2xx_t *dev, uint8_t *data, size_t len,
```

Comment:
Alternatively put `DEVELHELP` around it and keep it in? In case of removing, shouldn't you also remove the prototype in at86rf2xx.h?